Test-Source/Integration.Tests.ps1

```diff
@@ -85,8 +85,8 @@ Describe 'Use cases for this module' {
 
             # 2. Build and push image to latest tag, then grab it and see it's ok
             $result = Invoke-DockerBuild -Registry $localRegistryName -ImageName $imageName |
-                Invoke-DockerPush |
-                Invoke-DockerPull
+            Invoke-DockerPush |
+            Invoke-DockerPull
```

Comment:
This new alignment makes it slightly harder to notice that the 3 lines is 1 long piped command, until you really read it.